### PR TITLE
Bump version 4.2.7

### PR DESCRIPTION
**Pull request by Raudius**
## Release pull request
- [x] Pull latest stable branch `git checkout stableX`
- [x] Branch off a release branch `git checkout -b release/X.X.X`
- [x] Bump versions
  - [x] appinfo/info.xml
  - [x] package.json
  - [x] package-lock.json
- [x] Update changelog
- [x] Commit the version bump and push to origin 
- [ ] Merge release pull request

## Release the app

- [ ] Once merged, checkout the stable branch again `git checkout stableX` and `git pull`
- [ ] Tag a new release `git tag vx.x.x`
- [ ] Push tag
  - [ ] to the main repo: `git push origin stableX --tags`
  - [ ] to the release repo: `git push release stableX --tags`
- [ ] Build release on https://github.com/nextcloud-releases/richdocuments/tags
  - [ ] Draft a new release with the tag, add the changelog and publish
  - [ ] Wait a few minutes for the automation to build the app and to be published to the App Store

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 4.2.7
+
+- Disable remote image insertion on public links @juliushaertl
+- Fixed file revision history behaviour in group folders @Raudius
```
Comment (juliushaertl) [inline CHANGELOG.md:6]: I just merged the pending backport, so this needs a rebase and adding https://github.com/nextcloud/richdocuments/pull/2214 to the changelog :)

Comment (Raudius) [reply CHANGELOG.md:6]: Whoops, was a bit too eager to try out the release procedure :)

Rebasing done!

Comment (juliushaertl) [reply CHANGELOG.md:6]: No worries :) Just then we need also an entry on the changelog

Comment (Raudius) [reply CHANGELOG.md:6]: Fixed

**Review by juliushaertl — APPROVED** [review]

**Comment by juliushaertl** [comment]
@Raudius Seems the release at https://github.com/nextcloud-releases/richdocuments/releases is still missing ;)

**Pull request merged**